design/authn_azure/authn_azure_solution_design.md

```diff
@@ -420,44 +420,44 @@ Azure authenticator performance should conform with our other authenticators wit
-| 13-4 | A host with Subscription ID + Resource Group in its annotations                                                      | That host attempts to authenticates with authn-azure                | Return 200 status and `ValdatingRoleAnnotations` message appears in log                  | Integration               | [ ]     |
-| 13-5 | A host with Subscription ID OR Resource Group in its annotations                                                     | That host attempts to authenticates with authn-azure                | Return 403 Forbidden error and `RoleMissingAnnotations` message appears in log           | Integration               | [ ]     |
-| 13-6 | A host with Subscription ID + Resource Group + User Assigned Identity + System Assigned Identity in its annotations  | That host attempts to authenticates with authn-azure                | Return 403 Forbidden error and `IdentityAssignedInvalid` message appears in log          | Integration               | [ ]     |
+|      | Given                                                                                                                | When                                                                | Then                                                                                     | Manual / UT / Integration | Status                            |Core Test|
+|------|----------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------------------|------------------------------------------------------------------------------------------|---------------------------|-----------------------------------|----------
+| 13-1 | A host with no Azure-specific identification in its annotations or they are empty                                    | That host attempts to authenticates with authn-azure                | Return 403 Forbidden error and `RoleMissingAnnotations` message appears in log           | Integration               | <ul><li> [ ] Tested</li></ul>     |Yes      |
+| 13-2 | A host with Subscription ID + Resource Group + User Assigned Identity in its annotations                             | That host attempts to authenticates with authn-azure                | Return 200 status and `ValdatingRoleAnnotations` message appears in log                  | Integration               | <ul><li> [ ] Tested</li></ul>     |Yes      | 
+| 13-3 | A host with Subscription ID + Resource Group + System Assigned Identity in its annotations                           | That host attempts to authenticates with authn-azure                | Return 200 status and `ValdatingRoleAnnotations` message appears in log                  | Integration               | <ul><li> [ ] Tested</li></ul>     |No       |
```

Comment:
Let's make the When more specific - That host attempts to authenticates with authn-azure with an Azure token that has a System Assigned Identity

Reply:
Done